src/DynamoPackagesUI/ViewModels/PackageManagerViewModel.cs

```diff
@@ -16,36 +16,29 @@ namespace Dynamo.DynamoPackagesUI.ViewModels
     public class PackageManagerViewModel
     {
         public const string PACKAGE_MANAGER_URL = "http://dynamopackagemanager.com.s3-website-us-east-1.amazonaws.com";
+        
         public string Address { get; set; }
 
-        internal PackageManagerCefHelper CefHelper { get; set; }
+        internal PackageManagerCommands CefHelper { get; set; }
+
+        internal PublishCommands PublishCompCefHelper { get; set; }
 
-        internal PublishCefHelper PublishCompCefHelper { get; set; }
 
         /// <summary>
         /// Constructor
         /// </summary>
         /// <param name="dynamoViewModel"></param>
         /// <param name="model"></param>
         /// <param name="address"></param>
-        public PackageManagerViewModel(DynamoViewModel dynamoViewModel, PackageLoader model, string address)
+        public PackageManagerViewModel(DynamoViewModel dynamoViewModel, string address)
```

Comment:
I don't think that this constructor requires DynamoViewModel, you can directly use PackageLoader as a constructor parameter.